mocha-casperjs.js

```diff
@@ -48,7 +48,6 @@ module.exports = function (Mocha, casper, utils) {
   })
 
   casper.on('waitFor.timeout', function(timeout, details) {
-    casper.step++
```

Comment:
Yeah `timeout` is like a page timeout, not of the `waitFor` variety. It's a bit different.